odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -138,6 +139,11 @@ def tam_danielson_window(ray_trafo, smoothing_width=0.05, n_half_rot=1):
     *Exact cone beam CT with a spiral scan*.
     Physics in Medicine & Biology 4 (1998), p 1015.
     https://dx.doi.org/10.1088/0031-9155/43/4/028
+
+    [PKGT2000] Proksa R, Kohler T, Grass M, Timmer J.
```

Comment:
Disagree with replacing "Köhler" with "Kohler", it's wrong. Please use the unicode marker instead.

Reply:
You caught me :( I'll fix

Reply:
Don't mess with German names :wink: